include/client.h

```diff
@@ -11,6 +11,35 @@ namespace prism
 namespace connect
 {
 
+struct SdkVersion
+{
+    // ugly workaround for ugly bug in gcc, defining macros major() and minor()
+#undef major
+#undef minor
+
+    SdkVersion(uint8_t major, uint8_t minor, uint8_t revision)
```

Comment:
I would say revision as 8 bit int is not enough. We may have much more than 255. I would make revision 16 bit uint